config/docs.exs

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+
+# Configure ExDoc to use makeup for syntax highlighting
+config :ex_doc, :markdown_processor, Makedown.ExDoc
```

Comment:
I think you sent a PR for making this configurable on the project but we could not agree on it because it was global right? On the other hand, having to specify this on a separate file is very annoying, I am wondering if we should allow it to be specified in the project and document it is a global config.

Reply:
Yes, I did send such a PR. We can talk about it again on the ExDoc issue tracker.